misc/python/materialize/data_ingest/executor.py

```diff
@@ -133,6 +133,7 @@ def __init__(
             registry, json.dumps(key_schema), lambda d, ctx: d
         )
 
+        # TODO: Is this correct at all? Should this be self.topic? Check with def-
```

Comment:
Check with @def-

Reply:
I think those need to be under the ` for topic, f in fs.items():` block. That is, for every topic created we register the two schemas.

Reply:
Yes, probably. Let me double-check with Dennis.

=> I will move this to a follow-up PR not to block this one.